packages/core/src/zip-foundry.ts

```diff
@@ -0,0 +1,223 @@
+
+        case 'ERC1155':
+          return [
+            'function testUri() public {',
+            [
+              `assertEq(instance.uri(0), "${opts.uri}");`
+            ],
+            '}',
+          ];
+
+        case 'Governor':
+        case 'Custom':
+          return [
+            'function testSomething() public {',
+            [
+              '// Add your test here',
+            ],
+            '}',
+          ]
+
+        default:
+          throw new Error('Unknown ERC');
+      }
+    }
+    return [];
+  }
+};
+
+const script = (c: Contract) => {
+  return formatLinesWithSpaces(
+    2,
+    ...spaceBetween(
+      getHeader(c),
+      getImports(c),
+      getScript(c),
+    ),
+  );
+
+  function getImports(c: Contract) {
+    return [
+      'import "forge-std/Script.sol";',
+      `import "../src/${c.name}.sol";`,
+    ];
+  }
+
+  function getScript(c: Contract) {
+    return [
+      `contract ${c.name}Script is Script {`,
+      spaceBetween(
+        [
+          'function setUp() public {}',
+        ],
+        [
+          'function run() public {',
+          [
+            'vm.startBroadcast();',
+            `${c.name} instance = new ${c.name}();`,
+            'console.log("Contract deployed to %s", address(instance));',
+            'vm.stopBroadcast();',
+          ],
+          '}',
+        ],
+      ),
+      '}',
+    ];
+  }
+};
+
+const setupSh = `\
+# Check if git is installed
+if ! [ -x "$(command -v git)" ]
+then
+  echo "git command not found. Install git and try again."
+  exit 1
+fi
+
+# Check if Foundry is installed
+if ! [ -x "$(command -v forge)" ]
+then
+  echo "forge command not found. Install Foundry and try again. See https://book.getfoundry.sh/getting-started/installation"
+  exit 1
+fi
+
+# Setup Foundry project
+if ! [ -f "foundry.toml" ]
+then
+  echo "Initializing Foundry project..."
+
+  # Initialize sample Foundry project
+  forge init --force --no-commit --quiet
+
+  # Install OpenZeppelin Contracts
+  forge install OpenZeppelin/openzeppelin-contracts@v${contracts.version} --no-commit --quiet
+
+  # Remove template contracts
+  rm src/Counter.sol
+  rm script/Counter.s.sol
+  rm test/Counter.t.sol
+
+  # Add remappings
+  if [ -f "remappings.txt" ]
+  then
+    echo "" >> remappings.txt
+  fi
+  echo "@openzeppelin/=lib/openzeppelin-contracts/" >> remappings.txt
+
+  # Perform initial git commit
+  git add .
+  git commit -m "Initial commit" --quiet
+
+  echo "Done."
+else
+  echo "Foundry project already initialized."
+fi
+`;
+
+const readme = (c: Contract) => `\
+# Sample Foundry Project
+
+This project demonstrates a basic Foundry use case. It comes with a contract generated by [OpenZeppelin Wizard](https://wizard.openzeppelin.com/), a test for that contract, and a script that deploys that contract.
+
+## Installing Foundry
+
+See [Foundry installation guide](https://book.getfoundry.sh/getting-started/installation).
+
+## Initializing the project
+
+\`\`\`
+./setup.sh
```

Comment:
```suggestion
bash setup.sh
```

This works too and we wouldn't need to make the file executable. It feels somewhat cleaner to me.